lib/ronin/nmap/converters/csv.rb

```diff
@@ -102,7 +102,7 @@ def self.each_host_rows(host)
           ]
 
           host.each_port do |port|
-            yield(*host_row, *port_to_row(port))
+            yield([*host_row, *port_to_row(port)])
```

Comment:
Since we're yielding an Array instead of multiple arguments, we should also change the above code on line 78 which receives splatted arguments:

```ruby
each_host_rows(host) do |*row|
```

To

```ruby
each_host_rows(host) do |row|
```